src/main/kotlin/io/bazel/kotlin/builder/cmd/BUILD.bazel

```diff
@@ -1,5 +1,14 @@
 load("//src/main/kotlin:bootstrap.bzl", "kt_bootstrap_binary", "kt_bootstrap_library")
 
+_KOTLIN_STDLIBS = [
```

Comment:
this is repeated in `src/main/starlark/core/compile/cli/BUILD.bazel` below, is there a preferred place to consolidate these?

Reply:
[artifacts.bzl](https://github.com/bazelbuild/rules_kotlin/blob/master/src/main/starlark/core/repositories/kotlin/artifacts.bzl) is the best place for right now.

I've been moving deck chairs of late.

Reply:
Wait, wait. That will produce a cycle.

Reply:
[here should be safe](https://github.com/bazelbuild/rules_kotlin/blob/master/kotlin/compiler/compiler.bzl)

Reply:
thanks! moved